Makefile.vars.mk

```diff
@@ -28,8 +28,8 @@ UPBOUND_PACKAGE_IMG ?= $(UPBOUND_CONTAINER_REGISTRY)/$(PROJECT_OWNER)/$(PROJECT_
 # https://hub.docker.com/r/kindest/node/tags
 KIND_NODE_VERSION ?= v1.24.0
 KIND_IMAGE ?= docker.io/kindest/node:$(KIND_NODE_VERSION)
-KIND_KUBECONFIG ?= $(kind_dir)/kind-kubeconfig-$(KIND_NODE_VERSION)
-KIND_CLUSTER ?= $(PROJECT_NAME)-$(KIND_NODE_VERSION)
+KIND_KUBECONFIG ?= /home/wejdross/vshn/shedar/kindev/.kind/kind-kubeconfig-v1.24.4
+KIND_CLUSTER ?= kindev
```

Comment:
we should not name the kind cluster the same as kinddev.

Reply:
restored those files, merging